docs/intellicode-visual-studio-code-completions.md

```diff
@@ -0,0 +1,49 @@
+---
+title: VS Code Completions Preview
+description: Read an overview of IntelliCode for Visual Studio Code. The IntelliCode extension provides AI-assisted IntelliSense for Python, Java, TypeScript, and JavaScript.
+ms.custom: SEO-VS-2020
+ms.date: 04/19/2022
+ms.prod: visual-studio-family
+ms.technology: intellicode
+ms.topic: conceptual
+author: aayim
+ms.author: aayim
+manager: andster
+---
+
+# IntelliCode Completions Preview for Visual Studio Code
+
+The [IntelliCode Completions extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=VisualStudioExptTeam.vscodeintellicode-completions) is an experimental extension which predicts up to a whole line of code based on your current code context.
+
+![IntelliCode Completions for Python in Visual Studio Code](../images/wlc.gif)
+
+This extension may contain unstable, experimental code. The IntelliCode team is focused on iterating rapidly and new functionality will be added.
+
+### How to use
+
+![IntelliCode Completions interpreting IntelliSense completion list selection in Visual Studio Code](../images/intellisenseSelection.gif)
+
+Completions will appear after your cursor as you type, with a faded color. When the IntelliSense list is open, its selection is used to inform the whole-line autocompletion.
+
+At any time you can accept the suggestion by pressing the tab key (you will need to press it twice if the IntelliSense list is visible). If you don't want to accept the suggestion, you can just keep on typing or press ESC

```

Comment:
"you will need to press it twice, first to accept the IntelliSense selected, second to accept the IntelliCode whole line completions." 
^ To make the distinction that the users don't have to double click Tab

Reply:
New text: if the Intellisense list is visible, you will need to press TAB twice - once to accept the IntelliSense selection, and again to accept the rest of the whole-line autocompletion